site/content/in-dev/unreleased/getting-started/_index.md

```diff
@@ -23,19 +23,105 @@ type: docs
 weight: 101
 ---
 
-There are several options for getting started with Apache Polaris.
+# Getting Started with Apache Polaris Binary Distribution
 
-To quickly try out Apache Polaris, please use the [quickstart guide](./quick-start). For other examples, please see below.
+Quickly start Apache Polaris by running the pre-built binary, no build needed.
 
-## Docker Compose Examples
-Each of the proceeding Docker Compose examples provides a complete working environment with detailed instructions.
+---
+
+## Prerequisites
+
+- Java 21 or later installed. You can verify this by running:
+
+```bash
+java -version
+```
+
+---
```

Comment:
Nit: the separator may not be quite helpful for a small section like this. Could we remove all of them?